test/metabase/api/card_test.clj

```diff
@@ -597,14 +597,16 @@
                        mt/boolean-ids-and-timestamps
                        :last-edit-info)))))
         (testing "Card should include moderation reviews"
-          (letfn [(clean [mr] (select-keys [mr] [:status :text]))]
+          (letfn [(clean [mr] (-> mr
+                                  (update :user #(select-keys % [:id]))
+                                  (select-keys [:status :text :user])))]
             (mt/with-temp* [ModerationReview [review {:moderated_item_id (:id card)
                                                       :moderated_item_type "card"
                                                       :moderator_id (mt/user->id :rasta)
                                                       :most_recent true
                                                       :status "verified"
                                                       :text "lookin good"}]]
-              (is (= [(clean review)]
+              (is (= [(clean (assoc review :user {:id true}))]
```

Comment:
Is there an equivalent dashboard test?  It seems like you added the hydration for both card and dashboard, but only this test was modified.

Reply:
There isn't yet. The next step is to have verifications on dashboards and then it will get fleshed out more